File: HighlandsCore/src/main/java/highlands/highlandscore/commands/HighlandsCommandHandler.java
package highlands.highlandscore.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class HighlandsCommandHandler implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        sender.sendMessage(new String[]{ "Usage:", "/hlc" });
        return true;
    }
}

File: HighlandsCore/src/main/kotlin/com/highlands/highlandscore/core/commands/HighlandsCommandHandler.kt
package com.highlands.highlandscore.core.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class HighlandsCommandHandler : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        sender.sendMessage(arrayOf("Usage:", "/hlc"))
        return true
    }
}